packages/report-server/src/reportBuilder/transform/functions/mergeRows/getMergeStrategy.ts

```diff
@@ -19,40 +19,57 @@ export const buildGetMergeStrategy = (
   using?: keyof typeof mergeStrategies | Record<string, keyof typeof mergeStrategies>,
 ) => {
   const isGroupByField = buildIsGroupByField(groupBy);
+  const knownMergeStrategies: Record<string, keyof typeof mergeStrategies> = {};
+  const setAndReturnStrategy = (field: string, strategy: keyof typeof mergeStrategies) => {
+    knownMergeStrategies[field] = strategy;
+    return strategy;
+  };
```

Comment:
This optimisation is probably a bit unnecessary, but given that there's usually many many more rows than there are columns it would save a bit of time just to cache the known values for each column

Reply:
I like it :-)